.github/workflows/ci-build.yaml

```diff
@@ -70,9 +81,20 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Get pip cache dir
+      id: pip-cache
       run: |
         python -m pip install --upgrade pip
+        echo "::set-output name=dir::$(pip cache dir)"
+    - name: pip cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.pip-cache.outputs.dir }}
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py', '**/requirements.txt', '**/test-requirements.txt') }}
```

Comment:
Does the setup.py have to be part of the key?

Reply:
Since jax install requirements are listed in `setup.py`, I thought it would be useful to include it in the key.